pkgs/tools/package-management/rpm/default.nix

```diff
@@ -52,13 +53,20 @@ stdenv.mkDerivation rec {
     # symlinks produced by build are incorrect
     ln -sf $out/bin/{rpm,rpmquery}
     ln -sf $out/bin/{rpm,rpmverify}
+  '' + stdenv.lib.optionalString stdenv.isDarwin ''
+    cd $out/bin
+    ln -s "${nss}/lib/libnss3.dylib"
```

Comment:
I would prefer to fix the install_name of these libraries, everything that depends on them is going to have this problem otherwise.

Reply:
That sounds right. I wasn't sure if there is a reason for them to have this problem. New to Nix so didn't want to assume. Should they be their own issues / PR's?

Reply:
You can add it to this pr if you want, just try to keep the changes in a separate commit per package.